ui/app/src/assets/uiConfigDefaults.ts

```diff
@@ -48,7 +48,8 @@ const uiConfigDefaults = {
                 uiKey: count++,
                 configuration: {
                   title: {
-                    id: 'previewSearchPanel.title'
+                    id: 'previewSearchPanel.title',
```

Comment:
Are **all** these new translations (all on this file) already on a `defineMessges({...})` somewhere else? New translations that aren't already on other places may be declared on this file.